pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -358,54 +368,38 @@ func (db *DB) GetBandwidthAllocations() (map[storj.NodeID][]*Agreement, error) {
 
 	agreements := make(map[storj.NodeID][]*Agreement)
 	for rows.Next() {
-		rbaBytes := []byte{}
 		agreement := &Agreement{}
-		var satellite []byte
-		err := rows.Scan(&satellite, &rbaBytes)
+		satelliteID := storj.NodeID{}
+		err := rows.Scan(&satelliteID, &agreement.Agreement)
 		if err != nil {
 			return agreements, err
 		}
-		err = proto.Unmarshal(rbaBytes, &agreement.Agreement)
-		if err != nil {
-			return agreements, err
-		}
-		satelliteID, err := storj.NodeIDFromBytes(satellite)
-		if err != nil {
-			return nil, err
-		}
 		agreements[satelliteID] = append(agreements[satelliteID], agreement)
 	}
 	return agreements, nil
 }
 
 // GetBwaStatusBySerialNum get BWA status by serial num
 func (db *DB) GetBwaStatusBySerialNum(serialnum string) (status AgreementStatus, err error) {
-	defer db.locked()()
```

Comment:
It wouldn't been easier to make Lock and Unlock noop initially.

Reply:
yeah, but its already done :/